config/v1/types_scheduling.go

```diff
@@ -0,0 +1,35 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Scheduling holds cluster-wide information about Scheduling.  The canonical name is `cluster`
+type Scheduling struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	Spec SchedulingSpec `json:"spec"`
+	// status holds observed values from the cluster. They may not be overridden.
+	Status SchedulingStatus `json:"status"`
+}
+
+type SchedulingSpec struct {
+	// default node selector (I would be happy to see this die....)
```

Comment:
Node selector is the most useful security thing we have done so far, so while I know you hate it... :)

Reply:
> Node selector is the most useful security thing we have done so far, so while I know you hate it... :)

I hate that we have two incompatible implementations that have existed for years and never been cleaned up.

Reply:
Yeah, but both of those are global security config.  Either way we agree.